zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/impl/ProcessingStateMachine.java

```diff
@@ -652,13 +652,11 @@ private void updateState() {
         retryFuture,
         (bool, throwable) -> {
           if (throwable != null) {
-            LOG.error(ERROR_MESSAGE_UPDATE_STATE_FAILED, currentRecord, metadata, throwable);
```

Comment:
❓ I think we can delete the const `ERROR_MESSAGE_UPDATE_STATE_FAILED` then as well?